lib/galaxy/datatypes/hubAssembly.py

```diff
@@ -0,0 +1,59 @@
+"""
+HubAssembly datatype
+"""
+import logging
+import galaxy.version as version
+
+# Support for Galaxy <= 16.01
+if version.VERSION_MAJOR <= "16.01":
+    from galaxy.datatypes.images import Html
+else:
+    from galaxy.datatypes.text import Html
```

Comment:
Thanks for the explanation @jxtx. I got that point, and as I said, I had the case where I needed my datatype whereas I was not in 16.04.

But I guess this is a corner case, so I understand :).

Thanks for the support @jmchilton, I removed it as it seems to be a majority not in favor :).